microshift_running_apps/microshift-gitops.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-gitops"]
+= Automating application management with the GitOps agent
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-gitops
+
+toc::[]
+
+{microshift-short} GitOps is a lightweight, optional add-on agent derived from the Red Hat OpenShift GitOps Operator. GitOps for {microshift-short} uses the command-line interface (CLI) of Argo CD as the declarative GitOps engine. You can can consistently configure and deploy Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
+[id="microshift-gitops-can-do_{context}"]
+== What you can do with the GitOps agent
+By using the GitOps agent with {microshift-short}, you can leverage the following principles and tasks:
+
+* Extend the core principles of developing and maintaining software in a Git repository to the creation and management of your cluster and application configuration files. This means you can update the single repository and GitOps automates the deployment of new applications or updates to existing ones.
+** For example, if you have 1,000 edge devices, each using {microshift-short} and a local GitOps agent, you can easily add or update an application on all 1,000 devices with just one change in your central Git repository.
```

Comment:
I don't think these needs to be in a sub bullet, I think just a space would be good 
```suggestion

For example, if you have 1,000 edge devices, each using {microshift-short} and a local GitOps agent, you can easily add or update an application on all 1,000 devices with just one change in your central Git repository.
```

Reply:
Should we also mention that the changes to the Git Repository are controlled and governed by the PR approval process and all changes could be audited, (like who edited, who approved etc).